reference/orders.v2.oas2.yml

```
@@ -1278,6 +1426,8 @@ components:
         * `consignments` - include the response returned from the request to the `/orders/{order_id}/consignments` endpoint.
         
         * `consignments.line_items` - include the response returned from the request to the `/orders/{order_id}/products` endpoint in consignments. This implies `include=consignments`.
+
+        * `fees` - include the response returned from the request to the `/orders/{order_id}/fees` endpoint. This implies `include=fees`.
```

```suggestion
        * `fees` - include the response returned from the request to the `/orders/{order_id}/fees` endpoint.
```

I believe it's explicit in this case vs above where including line items will also include consignments.